packages/core/bootstrap/src/lib/cache-warmer/index.ts

```diff
@@ -19,53 +20,51 @@ interface WSInput {
   makeWSHandler?: MakeWSHandler
 }
 
-export const withCacheWarmer =
-  (warmerStore: Store<RootState>, middleware: Middleware[], ws: WSInput) =>
-  (rawExecute: Execute): Middleware =>
-  async (execute) =>
-  async (input: AdapterRequest) => {
-    const isWarmerActive =
-      util.parseBool(process.env.CACHE_ENABLED) &&
-      util.parseBool(process.env.EXPERIMENTAL_WARMUP_ENABLED)
-    if (!isWarmerActive) return await execute(input)
+export const withCacheWarmer = (
+  warmerStore: Store<RootState>,
+  middleware: Middleware[],
+  ws: WSInput,
+) => (rawExecute: Execute): Middleware => async (execute) => async (input: AdapterRequest) => {
+  const isWarmerActive =
+    util.parseBool(process.env.CACHE_ENABLED) &&
+    util.parseBool(process.env.EXPERIMENTAL_WARMUP_ENABLED)
+  if (!isWarmerActive) return await execute(input)
 
-    const wsConfig = getWSConfig()
-    const warmupSubscribedPayload = {
-      id: input.id,
-      // We need to initilialize the middleware on every beat to open a connection with the cache
-      // Wrapping `rawExecute` as `execute` is already wrapped with the default middleware. Warmer doesn't need every default middleware
-      executeFn: async (input: AdapterRequest) =>
-        await (
-          await withMiddleware(rawExecute, middleware)
-        )(input),
-      data: input,
-    }
-
-    if (wsConfig.enabled && ws.makeWSHandler) {
-      // If WS is available, and there is an active subscription, warmer should not be active
-      const wsHandler = await ws.makeWSHandler()
-      const wsSubscriptionKey = getSubsId(wsHandler.subscribe(input))
-      const cacheWarmerKey = getSubscriptionKey(warmupSubscribedPayload)
+  const wsConfig = getWSConfig()
+  const warmupSubscribedPayload = {
+    id: input.id,
+    // We need to initilialize the middleware on every beat to open a connection with the cache
+    // Wrapping `rawExecute` as `execute` is already wrapped with the default middleware. Warmer doesn't need every default middleware
+    executeFn: async (input: AdapterRequest) =>
+      await (await withMiddleware(rawExecute, middleware))(input),
+    data: omit(input, ['debug', 'rateLimitMaxAge']),
```

Comment:
Why do we omit this? Shouldn't the hashing function do this for us?

Reply:
I suspect that it `object-hash` only ignores keys that are at the top level, not nested keys.
I was seeing duplicate entries when `rateLimitMaxAge` changed within debug.

Reply:
Shouldn't we omit top level `debug` from `object-hash` anyway?

Reply:
Top level `debug` is already ignored. `excludeKeys` defaults to 
```
[
  'id',
  'maxAge',
  'meta',
  'debug',
  'rateLimitMaxAge',
]
```